lib/service_manager.rb

```
@@ -17,81 +17,257 @@ def self.stop(options)
+        
+        any_service_killed |= kill_php_fpm
+        any_service_killed |= kill_mysql
+        any_service_killed |= kill_redis
+        any_service_killed |= kill_memcached
+        any_service_killed |= kill_nginx
+        any_service_killed |= kill_apache
+        
+        if any_service_killed
+          puts "Forcible termination of services completed."
+        end
+      end
```

RuboCop (1.73)

[convention] Assignment Branch Condition size for kill_services is too high. [<26, 28, 14> 40.69/23]

(Metrics/AbcSize)